app/services/process_ssm.rb

```diff
@@ -0,0 +1,23 @@
+class ProcessSsm
```

Comment:
Since this class represents an SsmParameter, just call it that.

```suggestion
class SsmParameter
```